### ISSUE TITLE
[PRO]: Possible to change placeholder text programmatically after initial load

### ISSUE THREAD

**Issue by zzorba** [10play/10tap-editor#211]
**Describe the bug**
We are trying to find a way to change the Placeholder text programmatically after the editor has already loaded. It shouldn't clear the text in the editor, just change the default messages shown if the box is emptied.

It doesn't seem like this is possible at the moment (apologies if we missed it), is this something that could be added in a future release (Pro or otherwise)?

**Comment by GuySerfaty** [comment]
Hi! Update an extension configuration after initialization looks hard, but we got a couple of requests for that feature and we are going to keep it as a free feature :)

here is the PR, we will try to merge soon:

https://github.com/10play/10tap-editor/pull/213

**Comment by zzorba** [comment]
@GuySerfaty I tried out the latest release (05.23), and I see the new `setPlaceholder` method on the editor bridge.  But for some reason, it always appears to be undefined when I try to call it?

I rebuild my embedded editor and everything appears to type-check, but it fails at runtime.  Any thoughts?

```
  const editor = useEditorBridge({
    // DO NOT USE avoidIosKeyboard, it is a trap.
    dynamicHeight: true,
    bridgeExtensions: [
      ...TenTapStartKit,
      CoreBridge.configureCSS(editorCss),
      PlaceholderBridge.configureExtension({
        placeholder: placeholder ?? t`Send a message...`,
      }),
      ...
```

Here is how I'm initializing my bridge.

**Comment by zzorba** [comment]
Thank you so much, I see this is now fixed and released in 0.5.23 and its working great.
